config/config.h.in

```diff
@@ -57,5 +57,7 @@
 
 #cmakedefine HAVE_BE64TOH
 #cmakedefine ORDER_BIG_ENDIAN
+#cmakedefine HAVE_ENDIAN_H
+#cmakedefine HAVE_SYS_ENDIAN_H
```

Comment:
Maybe better `#define ENDIAN_INCLUDE_FILE <@ENDIAN_INCLUDE_FILE@>` and `# include ENDIAN_INCLUDE_FILE` in `shada.c`?